DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -560,6 +560,42 @@ DDFilteredView::nav_type DDFilteredView::navPos() const {
   return pos;
 }
 
+const int DDFilteredView::level() const {
```

Comment:
@Ianna, Will this conflict with https://github.com/cms-sw/cmssw/pull/31482/files#diff-06d03fa40cda4df4cac28c118cada0eaR563?

Reply:
No worries, I love merging conflicts :-)